packages/babel-plugin-transform-async-generator-functions/test/fixtures/for-await/async-generator-exec/exec.js

```diff
@@ -22,5 +24,7 @@ var output = 0;
 return forEach(genAnswers(), function(val) { output += val.value })
 .then(function () {
   assert.equal(output, 42);
+})
+.catch(function (error) {
```

Comment:
@alxpy the original issue was asking to add support for failing an async test via rejected promise to `fixture-test-runner` since @hzoo added support for [returning a promise](https://github.com/babel/babel/blob/master/packages/babel-helper-transform-fixture-test-runner/src/index.js#L186).

ref: [original discussion](https://github.com/babel/babel/pull/4576/files#r80736933)